lib/chef/provisioning/machine/unix_machine.rb

```diff
@@ -223,6 +223,10 @@ def detect_sh
 platform="aix"
 platform_version=`uname -v`
 machine="ppc"
+# Linux supporting /etc/os-release
```

Comment:
Is this also in the main https://chef.io/install.sh?  I'd like to make sure these two scripts stay in sync.